content/templates/track.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block content %}
+<h1>{{ object.artist }}</h1>
```

Comment:
I think this should be `object.title`, as the  track object does not have an artist field (only the contributors you're displaying below). That's why this is not showing up on the page.